bubblewrap.c

```diff
@@ -2503,6 +2531,10 @@ parse_args_recurse (int          *argcp,
           argc -= 1;
           break;
         }
+      else if (strcmp (arg, "--no-int-term") == 0)
```

Comment:
To me, `--forward-signals` would seem like a better name for this. (Or maybe `--propagate-signals`, but you've already demonstrated that "propagate" is harder to spell!)